parlai/core/loader.py

```diff
@@ -361,12 +361,7 @@ def load_world_module(
         repo = 'parlai_internal'
         task = task[9:]
     task_path_list = task.split(':')
-    if '.' in task_path_list[0]:
-        # The case of opt['task'] = 'parlai.tasks.squad.agents:DefaultTeacher'
-        # (i.e. specifying your own path directly, assumes DialogPartnerWorld)
-        return _get_default_world(default_world, num_agents)
 
-    task = task_path_list[0].lower()
```

Comment:
Wtf why did we lower it

Reply:
Convention, probably? Namely, none of our task folder names (`convai2`, `blended_skill_talk`, etc.) are capitalized